EstFoisy/theia
# Corrected issue regarding browser validation message for float number inputs

#### What it does
For the preference number input if a float should be accepted in that field there will be no more a failure message showing.
Closes #11851

#### How to test
1.	Go into file: preference: Open Setting
2.	Find a field that should accept a float value
3.	Enter a float value
4.	There should be no message appearing
5.	Try a field that should only take an integer 
6.	Enter a float value
7.	The failure message regarding the value being incorrect should appear


https://user-images.githubusercontent.com/113064863/203114412-6758865e-a1c7-4be3-98e0-6402ee440746.mp4



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
